notebooks/14_regression/index.md

```diff
@@ -0,0 +1,222 @@
+# Regression
```

Comment:
Title of the page should be in another format. Change it and put the name of authors inside, too.

Reply:
I'm not sure how it should be. Comparing with other notebooks, the title seems fine. @nimajam41